cloud-src/modules/ROOT/pages/user-management/organization.adoc

```diff
@@ -0,0 +1,85 @@
+= Organizations
+:keywords: typedb, cloud, organization
+:longTailKeywords: typedb cloud organization, organization management, create organization
+:pageTitle: Organizations
+:summary: Organization management details.
+:experimental:
+
+// tag::intro[]
+An organization combines all users and resources (deployments) of your company, as well as Projects (to group resources)
+and Teams (to group users).
+
+By default, all users are created as the only member of their personal organization, named after their User ID.
+// end::intro[]
+
+[#_select]
+== Select an active organization
+
+// tag::select_organization[]
+A user can be a member of multiple organizations.
+
+To select an active one to work with at the moment, use the drop-down menu on top of the main navigation menu on the
+left edge.
+// end::select_organization[]
+
+== Create a new organization
+
+// tag::create_organization[]
+To create a new organization open settings by clicking on the gear (⚙) icon in the top right corner of any page of
+TypeDB Cloud https://cloud.typedb.com[web portal].
+
+In the Settings page click the btn:[New organization] button, set the new organization's visible name and unique id, and
+click btn:[Submit].
+// end::create_organization[]
+
+== Delete an organization
+
+Open the xref:cloud::user-management/organization.adoc#_show[Organization details] page and click the
+btn:[Delete organization] button at the end.
+
+[NOTE]
+====
+To delete an organization there needs to be no Projects added to it.
+====
+
+== Join an organization
+
+// tag::join_organization[]
+The only way to join an existing organization is to be invited by one of its members.
```

Comment:
Currently, we do not invite users to join an organization or team. When a user is added to an organization or team, the user immediately becomes a member. This is different from inviting, as the user cannot reject the invitation.

Reply:
> Currently, we do not invite users to join an organization or team. When a user is added to an organization or team, the user immediately becomes a member. This is different from inviting, as the user cannot reject the invitation.

I know that. 
It was named that way deliberately, partially because I didn't find a term for how to call that mandatory invitation with auto-acceptance. And I was under the impression that the only way to proceed is to implement a mechanism to accept or deny those invitations.